.ci/script.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
 CI_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=/dev/null
 source "${CI_DIR}/common.sh"
 
 travis_retry make fetchthirdparty
+find . -type f -name '*.sh' -not -path "./base/*" -not -path "./luajit-rocks/*" -print0 | xargs --null shellcheck
```

Comment:
It occurs to me that it might be better to grep for a few relevant shebangs as well. See, e.g., [here](https://github.com/travis-ci/packer-templates/blob/d909937720520720d93f1f7f454638e7ed8b71d1/runtests) which I came across while searching if there was a precompiled updated shellcheck.

```bash
echo 'Running shellcheck'
git grep -l '^#!/usr/bin/env bash' | xargs shellcheck

echo 'Running shfmt'
git grep -l '^#!/usr/bin/env bash' | xargs shfmt -i 2 -w
```